lib/logstash/outputs/opensearch.rb

```diff
@@ -128,7 +137,7 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
 
   config :document_type,
     :validate => :string,
-    :deprecated => "Document types are being deprecated in Elasticsearch 6.0, and removed entirely in 7.0. You should avoid this feature"
+    :deprecated => "Document types are being deprecated in OpenSearch 6.0, and removed entirely in 7.0. You should avoid this feature"
```

Comment:
Is this line still necessary?

Reply:
Good catch. I will remove 6.0 . In this PR i am not removing any code which is being used by ES 7.10.2. I will do it next PR. What do you think?

Reply:
Sounds good